css/style.css

```diff
@@ -1,18 +1,184 @@
 * {
   box-sizing: border-box;
   max-width: 1440px;
-  margin-left: auto;
-  margin-right: auto;
+  margin: 0 auto;
   font-family: "Roboto", sans-serif;
   font-style: normal;
 }
 
 body {
   background: #fdfdfd;
```

Comment:
If you need only color, you should use `background-color`, instead of `background`
Because `background` will add a lot of excess info
<img width="337" alt="Screenshot 2022-03-15 at 18 11 23" src="https://user-images.githubusercontent.com/25488411/158409524-ac90dc27-2706-4777-b48e-974f9a8ea203.png">